lib/server/endpoint.ex

```diff
@@ -1,7 +1,7 @@
-defmodule Server.Endpoint do
+defmodule KinoLiveViewNative.Server.Endpoint do
```

Comment:
I'd like this change reverted. This library is intended to be used within a Livebook and there is no chance of a name collision with another app called `Server`

Reply:
Done